src/app/[locale]/_component/profileCards/profileCards.tsx

```diff
@@ -49,15 +50,11 @@ const ProfileCards = (): React.ReactElement => {
               <Typography variant="body2" py={2}>
                 {t('common.lockappaccess')}
               </Typography>
-
-              <ButtonNaked
-                href={ROUTES.PROFILE_BLOCK}
-                color="primary"
-                endIcon={<ArrowForwardIcon />}
-                size="medium"
-              >
-                {t('profile.lockaccess')}
-              </ButtonNaked>
+              <Link href={ROUTES.PROFILE_BLOCK}>
```

Comment:
Here and in the other occurrences we can eliminate this link that wraps the button and directly insert the onclick on the button with the router.push (so, using the userRouter)